apartmenttherapy/oxygen-lib-ui
# Henry/email signup continuity

#### What does this PR do & why?
Along with some slight markup changes, this PR mainly flattens the props for `EmailSignup`  to match the proposed GraphQL schema.

#### How can this be manually tested?
You will need the companion Oxygen branch: https://github.com/apartmenttherapy/oxygen/pull/1008
Take a look at `/email` to see the revised component in action.

If you want to see it in a post, you can use `oxygen-gql` locally: https://github.com/apartmenttherapy/oxygen-gql/pull/115

#### Any questions or concerns about this change?


#### Deployment Notes

_(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)_

#### Link to story card and other documentation
https://app.asana.com/0/937985426519793/951949451644305/f

#### GIF

#### Have you remembered to...?
